code/addons/actions/template/stories/basics.stories.tsx

```diff
@@ -1,17 +1,10 @@
-// TODO -- for now react, going to generalise
-import React, { FC } from 'react';
-
 import { action } from '@storybook/addon-actions';
 
-// TODO -- this needs to be a generic component
-const Button: FC<{ onClick: () => void }> = ({ onClick, children }) => (
-  <button type="button" onClick={onClick}>
-    {children}
-  </button>
-);
-
 export default {
-  component: Button,
+  component: 'Button',
```

Comment:
@tmeasday is this a typo? shouldn't it be `global.Components.Button`?

Reply:
Must be some kind of merging/updating snafu. I thought I checked it was working before I pushed but I guess not. It definitely said that at one point ;)